ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -266,7 +268,7 @@ public struct FluentNotification: View, ConfigurableTokenizedControl {
                 GeometryReader { proxy in
                     let proposedSize = proxy.size
                     let proposedWidth = proposedSize.width
-                    let horizontalPadding = 2 * tokens.presentationOffset
+                    let horizontalPadding = 2 * tokenSet[.presentationOffset].float
```

Comment:
Super tiny nit: consider adding a comment for why this is 2* here, instead of the offset just being the correct size?

Reply:
Sure! I will first have to figure out why it's 2* 😀

Reply:
I suspect 2* to account for the padding on either side

Reply:
Nope, I'm not gonna touch it because I don't know why it is that way, and I would prefer not to guess wrong. But if anyone else does know, please feel free to add some comments when you get a chance!

Reply:
Totally fair!